src/Composer/ScriptHandler.php

```diff
@@ -15,7 +15,7 @@ class ScriptHandler extends BaseScriptHandler
     /**
      * @param \Composer\Script\CommandEvent $event
```

Comment:
Can you update the phpDocumentor annotation as well? Apart from that, the PR is ok for me.